evil-collection.el

```diff
@@ -110,7 +110,7 @@ through removing their entry from `evil-collection-mode-list'."
   :type 'boolean
   :group 'evil-collection)
 
-(defcustom evil-collection-mode-list
+(defvar evil-collection--supported-modes
   `(2048-game
```

Comment:
Side comment @justbur . I'm not sure the best way to update the documentation / notes / comments for the following other places where evil-collection-mode-list is used.

```
ag -ri "evil-collection-mode-list"
modes/lispy/evil-collection-lispy.el
29:;; Add `lispy' to `evil-collection-mode-list' to use this package.
30:;; e.g. (push 'lispy evil-collection-mode-list)

evil-collection-settings.el
45:through removing their entry from `evil-collection-mode-list'."

evil-collection.el
104:through removing their entry from `evil-collection-mode-list'."
254:(defcustom evil-collection-mode-list evil-collection--supported-modes
567:  "Register the Evil bindings for all modes in `evil-collection-mode-list'.
576:instead of the modes in `evil-collection-mode-list'."
580:    (setq modes evil-collection-mode-list))

readme.org
50:   The list of supported modes is configured by ~evil-collection-mode-list~.
88:   NOTE: If you don't like surprises but still want to use ~evil-collection-init~, setting ~evil-collection-mode-list~ to nil
93:   Modify ~evil-collection-mode-list~ to disable or add any modes that should be evilified by ~evil-collection~.
438:   all modes in ~evil-collection-mode-list~:
```

Reply:
@justbur I was wondering if any of the documentation / notes / comments will need to be updated to evil-collection--supported-modes instead of evil-collection-mode-list ? I understand that evil-collection-mode-list remains functionally the same, and it calls upon evil-collection--supported-modes. 

The above list was just a project wide search for any instance of where the word "evil-collection-mode-list" was used with the [ag silver searcher utility](https://github.com/ggreer/the_silver_searcher).

Reply:
I don't think it's necessary. I just needed a temporary variable to set up the autoloads, because if someone customizes the mode-list variable I wouldn't have an exhaustive list to work with.

Reply:
@justbur ok. Just wanted to make sure.